templates/projects/snippets/relevancy.html

```diff
@@ -0,0 +1,96 @@
+<div id="task-app">
+  <task-presenter>
+  <div class="container-fluid">
+      <div class="row">
+        <blockquote>
+
+        <h3>Rating Topic Relevancy</h3>
+        <p>This template may be used to rate the relevancy of queries against a topic. The user is presented with a list of queries and can choose "relevant" or "irrelevant" with regard to the topic.</p>
+
+        <h4>Input</h4>
+        <p>Tasks can be imported by creating a CSV <a href="https://bbgithub.dev.bloomberg.com/gist/kbecker42/c3f2c64297169023d6126fa306e8b507" target="_blank">file</a> containing one column for the topic and a variable number of columns for queries.</p>
```

Comment:
Maybe its better to have this sample in the GIGwork docs instead of your personal bbgithub? What do you think ? And this comment apply for the other file that is pointing to your bbgithub

Reply:
Yes, that would work. Which specific location were you recommending to copy the files to? I can move them there. Thank you.

Reply:
Actually now that Im thinking about where, I dont think we should have any link pointing to an internal site since this template is going to be on public gigwork too. I think that those details should be part of gigwork documentation itself.

Reply:
I think Tim or Christoh should decide where they want  to have more details about how to use the template.

Reply:
Agreed. Links removed via https://github.com/bloomberg/pybossa-default-theme/pull/171/commits/c696c4aefcbc5a44ab72ae2cb8e5366be1900365